src/main/java/com/anonymous/privacytool/repository/AuthUserRepository.java

```diff
@@ -1,13 +1,15 @@
 package com.anonymous.privacytool.repository;
 
-import com.anonymous.privacytool.entity.AuthUser;
+import com.anonymous.privacytool.entity.User;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
 import java.util.Optional;
 import java.util.UUID;
 
 @Repository
-public interface AuthUserRepository extends JpaRepository<AuthUser, UUID> {
-    Optional<AuthUser> findByEmail(String email);
+public interface AuthUserRepository extends JpaRepository<User, UUID> {
+    Optional<User> findByEmail(String email);
+
+    Optional<User> findByEmailAndPassword(String email,String password);
```

Comment:
Consider fetching user & validating afterwards - will give visiblity if user has not signed up or password is wrong.